src/tile-board/Tileboard.jsx

```diff
@@ -78,4 +78,29 @@ function toggle(arrayGrid, y, x, size) {
   return tempArray;
 }
 
+/**
+ * Generates a random board
+ */
+function randomBoard(arrayGrid, size) {
+  let tempArray = arrayGrid.map((row) => {
+    return [...row];
+  });
+
+  for (let i = 0; i < size * size; i++) {
+    const x = randomTilePosition(size);
+    const y = randomTilePosition(size);
+
+    tempArray = toggle(tempArray, y, x, size);
+  }
+
+  return tempArray;
+}
```

Comment:
Personal preference: I like to keep these kind of functions in a separate `utils.js` or `functions.js` file and let my jsx files focus primarily on rendering elements and doing React stuff.

Reply:
Totally makes sense